src/ophyd_async/tango/base_devices/base_device.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import annotations
+
+import asyncio
+from typing import Optional
+
+from ophyd_async.core import (
+    DEFAULT_TIMEOUT,
+    Signal,
+    SignalR,
+    SignalRW,
+    SignalW,
+    SignalX,
+    StandardReadable,
+)
+from ophyd_async.tango.signal import (
+    infer_python_type,
+    infer_signal_frontend,
+    make_backend,
+)
+from tango.asyncio import DeviceProxy
+
+__all__ = ("TangoReadableDevice",)
+
+
+# --------------------------------------------------------------------
+class TangoReadableDevice(StandardReadable):
```

Comment:
I'm wondering if it makes sense to make a `class TangoDevice(Device)`, then have `class TangoMover(TangoDevice, StandardReadable)` which would allow you to make classes that are not `StandardReadable`s (maybe like LIMA plugins?) where they are intended to be children of `StandardDetector` rather than being read in a standalone way?

Reply:
This makes sense. As suggested I created `TangoDevice(Device)` which contains most of the cost that was previously in `TangoReadableDevice`. `TangoReadableDevice` was redefined to simply inherit from TangoDevice and StandardReadable so that it calls the TangoDevice init method.